roles/locale/tasks/el9.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install language packs.
```

Comment:
So this is new role that was created specifically for rhel9 and it installs a language pack - why is this needed?

Reply:
It installs amongst others `en_US.UTF-8`, which was present by default in CentOS 7 and is required for creating the Pulp PostgreSQL database, which wants that `locale` setting. I foresee a lot of other software that defaults to `en_US.UTF-8` and will not work properly without it.